src/Misc/layoutbin/installdependencies.sh

```diff
@@ -63,7 +63,12 @@ then
             
             # ubuntu 18 uses libcurl4
             # ubuntu 14, 16 and other linux use libcurl3
-            apt install -y libcurl3 || apt install -y libcurl4
+            if [ $(lsb_release -rs) = "18.04" ]
```

Comment:
Should this check for 14 and 16 instead because future versions are going to use 4?

Reply:
There's a workaround btw.  If you install libcurl4 first, then do:  apt-mark hold libcurl4
That will prevent the crapfest that follows.

Reply:
Also, why not just reverse it?  Otherwise you're going to have to continually keep adding future versions:

Reply:
updated the code to reflect the comments.